### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by andrearizzello**
### Changes
- Fixed typo errors in `DEVELOPERS.md`

### Related Issues
- Issue #115

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -24,28 +24,28 @@ machine:
...
-### Building Cicero
+### Building Concerto
 
-To build Cicero, you clone the source code repository and use lerna to build:
+To build Concerto, you clone the source code repository and use lerna to build:
 
 ```shell
 # Clone your Github repository:
-git clone https://github.com/<github username>/cicero.git
+git clone https://github.com/<github username>/Concerto.git
```
Comment (jeromesimeon) [inline DEVELOPERS.md]: This should be in lowercase (repository name).

File: DEVELOPERS.md [hunk 2]
```diff
@@ -24,28 +24,28 @@ machine:
...
 
-# Go to the Cicero directory:
-cd cicero
+# Go to the Concerto directory:
+cd Concerto
 
-# Add the main Cicero repository as an upstream remote to your repository:
-git remote add upstream "https://github.com/acccordproject/cicero.git"
+# Add the main Concerto repository as an upstream remote to your repository:
+git remote add upstream "https://github.com/accordproject/Concerto.git"
```
Comment (jeromesimeon) [inline DEVELOPERS.md]: This should be in lowercase (repository name).

File: DEVELOPERS.md [hunk 3]
```diff
@@ -24,28 +24,28 @@ machine:
...
 
 ```shell
 # Clone your Github repository:
-git clone https://github.com/<github username>/cicero.git
+git clone https://github.com/<github username>/Concerto.git
 
-# Go to the Cicero directory:
-cd cicero
+# Go to the Concerto directory:
+cd Concerto
```
Comment (jeromesimeon) [inline DEVELOPERS.md]: This should be in lowercase (repository name).

File: DEVELOPERS.md (hunk 1 above)
Comment (andrearizzello) [reply DEVELOPERS.md]: changed

File: DEVELOPERS.md (hunk 2 above)
Comment (andrearizzello) [reply DEVELOPERS.md]: changed

File: DEVELOPERS.md (hunk 3 above)
Comment (andrearizzello) [reply DEVELOPERS.md]: changed

**Comment by andrearizzello** [comment]
@jeromesimeon i updated the requested fields :)

**Pull request merged**